<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>321</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Исходная матрица</string>
     </property>
     <widget class="QTableWidget" name="matrixTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>321</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Результат запроса</string>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Выберите запрос</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Середнє арифметичне й середнє геометричне мінімальних  елементів у заданому рядку й стовпці.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номер першого зі стовпців, що містять хоча б один нульовий елемент.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Характеристикою рядка цілочисельної  матриці назвемо суму її негативних парних елементів. Визначити мінімальну й максимальну з характеристик.</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="queryRequestPushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="queryTextBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1003</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Входные данные</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Размерность матрицы</string>
     </property>
     <widget class="QPushButton" name="sizeOfMatrixApplyPushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите размерность   --&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rowsCountLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Данные</string>
     </property>
     <widget class="QPushButton" name="addNewValueToArrPushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="indexLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер элемента [][]  --&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
